client/i18n/locales/english/intro.json

```diff
@@ -2140,11 +2246,21 @@
       },
       "muyw": { "title": "266", "intro": [] },
       "mvzb": { "title": "267", "intro": [] },
-      "rmpy": { "title": "268", "intro": [] },
+      "quiz-javascript-data-structures": {
+        "title": "JavaScript Data Structures Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on JavaScript Data Structures."
+        ]
+      },
       "plfo": { "title": "269", "intro": [] },
       "xdyh": { "title": "270", "intro": [] },
       "ybrh": { "title": "271", "intro": [] },
-      "vjgg": { "title": "272", "intro": [] },
+      "quiz-javascript-graphs-and-trees": {
+        "title": "JavaScript Graphs and Trees Quiz",
```

Comment:
we could drop the javascript part from the titles and dashed names.
because this is more universal cs topics instead of js specific

Reply:
✅ Will be added with next commit.